packages/expo-web-browser/android/src/main/java/expo/modules/webbrowser/InternalCustomTabsActivitiesHelper.kt

```diff
@@ -0,0 +1,96 @@
+package expo.modules.webbrowser
+
+import android.app.Activity
+import expo.modules.webbrowser.error.PackageManagerNotFoundException
+import expo.modules.core.errors.CurrentActivityNotFoundException
+import android.content.pm.ResolveInfo
+import android.content.Intent
+import android.content.pm.PackageManager
+import android.net.Uri
+import androidx.browser.customtabs.CustomTabsClient
+import androidx.browser.customtabs.CustomTabsIntent
+import androidx.browser.customtabs.CustomTabsService
+import expo.modules.core.ModuleRegistry
+import expo.modules.core.interfaces.ActivityProvider
+import java.util.ArrayList
+import java.util.LinkedHashSet
+
+private const val DUMMY_URL = "https://expo.dev"
+
+internal class InternalCustomTabsActivitiesHelper : CustomTabsActivitiesHelper {
+  private lateinit var moduleRegistry: ModuleRegistry
+
+  override fun onCreate(moduleRegistry: ModuleRegistry) {
+    this.moduleRegistry = moduleRegistry
+  }
+  override fun onDestroy() = Unit
```

Comment:
Do you need it at all? 🤔

Reply:
Yes, the `CustomTabsActivitiesHelper` implements unimodules `InternalModule` which requires them all. But I'll get rid of them anyway when migrating to expo-module